docs/using/security-non-events.md

```diff
@@ -0,0 +1,58 @@
+* [CVE-2015-9037](http://www.cvedetails.com/cve/CVE-2014-9037/): Bug in authentication code could allow hijacking long-dormant accounts that used an older password hash algorithm. Sandstorm unaffected because Wordpress on Sandstorm relies on Sandstorm for authentication.
+* [CVE-2014-9033](http://www.cvedetails.com/cve/CVE-2014-9033/): CSRF in password reset flow. Wordpress on Sandstorm not vulnerable because it relies on Sandstorm for authentication and does not implement its own passwords.
+* [CVE-2014-5205](http://www.cvedetails.com/cve/CVE-2014-5205/), [5204](http://www.cvedetails.com/cve/CVE-2014-5204/): Possible bypass of CSRF protections. Sandstorm adds additional protection by putting the admin interface on a randomized hostname.
+* [CVE-2014-5203](http://www.cvedetails.com/cve/CVE-2014-5203/): Unauthenticated attacker can cause arbitrary code execution via malicious serialized data. On Sansdtorm, only editors/admins could exploit this.
+* [CVE-2014-0166](http://www.cvedetails.com/cve/CVE-2014-0166/): Possible to forge authentication cookies. Wordpress on Sandstorm not vulnerable because it relies on Sandstorm (not cookies) for authentication.
+
+## Roundcube
+
+* [CVE-2015-5381](https://security-tracker.debian.org/tracker/CVE-2015-5381): Reflected cross-site scripting. Sandstorm mitigates reflected-XSS attacks in much the same way it mitigates CSRF attacks: the attacker would have to know the app's randomly-generated hostname for the attack to work. An attacker with passive network MITM abilities or some other way of obtaining the hostname might be able to carry out an attack, but this is a much higher bar than outside of Sandstorm.
+* [CVE-2015-5382](https://security-tracker.debian.org/tracker/CVE-2015-5382): Reading arbitrary files on the filesystem. Much like with similar Etherpad issues, this is irrelevant on Sandstorm because each Roundcube mailbox runs as a separate instance of the app in an isolated container, these instances cannot see each other's files, and there is no way to send requests to a particular mailbox instance unless the owner has explicitly shared access with you.
+* [CVE-2015-5385](https://security-tracker.debian.org/tracker/CVE-2015-5383): Log data leakage. Irrelevant on Sandstorm because each Roundcube mailbox has a separate log file visible only to it.
+
+## Linux kernel
+
+The Linux kernel has had many bugs that could allow any local process to gain root privileges or otherwise bypass security rules. Sandstorm blocks most of these vulnerabilities by virtue of its [server-side sandbox](https://docs.sandstorm.io/en/latest/developing/security-practices/#server-sandboxing) which disables much of the Linux kernel API thereby reducing the surface of attack only to core, well-reviewed functionality.
```

Comment:
Best to link to the `*.md` files not the URL of the rendered doc. That results in local `mkdocs serve` users getting links to the local versions of the documents, and has other similar benefits.

I can submit patches to do that in a follow-up unless you find it particularly exciting to make that change.